chromeos.kernelci.org

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+# Periodic job run on staging.kernelci.org to merge together all the pending
+# PRs and update the staging.kernelci.org branches, then update Docker images
+# and trigger a full build/test cycle with a kernel branch based on linux-next.
+
+set -e
+
+CHROMEOS_SETTINGS="data/chromeos.ini"
+
+cmd_pull() {
+    echo "Updating local repository"
+    git pull --ff-only
+}
+
+cmd_jenkins() {
+    echo "Updating Jenkins jobs"
+    ./pending.py kernelci-jenkins --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+}
+
+cmd_core() {
+    echo "Updating kernelci-core"
+    ./pending.py kernelci-core --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+}
+
+cmd_test_definitions() {
+    echo "Updating test-definitions"
+    ./pending.py test-definitions --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+}
+
+cmd_backend() {
+    echo "Updating kernelci-backend"
+    ./pending.py kernelci-backend --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+    # ToDo: check if there were any changes to skip if there weren't any
+    ./ansible kernelci-backend api.chromeos.kernelci.org
+
+    echo "Resetting and enabling the Mongo DB profiler"
+    echo "db.setProfilingLevel(0); db.system.profile.drop(); db.setProfilingLevel(2);" | mongo chromeos-kernel-ci
```

Comment:
I still think we can drop this part, about Mongo DB profiler.  It's only really useful when changing the schema and adjusting indexes to ensure Mongo doesn't do full collection scans.

Reply:
done